llvm/test/CodeGen/X86/domain-reassignment.mir

```diff
@@ -860,3 +864,63 @@ body:             |
     RET 0
 
 ...
+---
+name:            test_globalisel_unused
```

Comment:
Why is the function named `*globalisel*`? It seems that there is not global isel flag in this test.

Reply:
Even if it is present, it has no impact, as we run a single pass and MIR has been already selected. I want to show here that MIR for this function is obtained using GlobalISel, as a result we have registers without classes `{ id: 0, class: _, preferred-register: '' }`. Yeah, I think more detailed comment will be better. Or do you think it may be even better to extract it to a separate file?